cogs/util/tmp_channel_util.py

```diff
@@ -244,63 +246,72 @@ def create_token() -> int:
         return int(totp.now())
 
     @staticmethod
-    async def check_delete_channel(voice_channel: VoiceChannel, db: TempChannels,
+    async def check_delete_channel(voice_channel: Optional[VoiceChannel], db: TempChannels,
                                    reset_delete_at: tuple[bool, PrimitiveMongoData] = (False, None)) -> bool:
 
+        try:
+            if len({member for member in voice_channel.members if not member.bot}) != 0:
+                return False
+
+            document: TempChannel = await db.find_one({DBKeyWrapperEnum.VOICE.value: voice_channel.id})
+        except AttributeError:
+            return True
+
         if len({member for member in voice_channel.members if not member.bot}) != 0:
             return False
```

Comment:
shoudn't this one be removed since it is done in the try block?

Reply:
yup you're right.